src/resources/views/form_content.blade.php

```diff
@@ -2,8 +2,13 @@
 <input type="hidden" name="locale" value={{ $crud->request->input('locale')?$crud->request->input('locale'):App::getLocale() }}>
 @endif
 
-{{-- See if we're using tabs --}}
-@if ($crud->tabsEnabled())
+@if($crud->custom_form_view)
+{{-- if we are using a custom form view extract template name and field for the programmer --}}
+    @include($crud->custom_form_view,[
+      't'=>$crud->extractTemplates($fields),
```

Comment:
Ok, humour me - what is `t` and `f` here?

Reply:
based on looking at the file, its an array where `t` is short fort `templates` and `f` is short for `fields`.